src/main/java/org/openml/webapplication/Main.java

```diff
@@ -117,21 +118,20 @@ public static void main( String[] args ) {
 					// bootstrap process dataset
 					String processMode = cli.hasOption("x") ? "random" : "normal";
 					new ProcessDataset(apiconnector, id, processMode);
-				} else if( function.equals("extract_features_all") ) {
 					
+				} else if( function.equals("extract_features_all") ) {
```

Comment:
what happened to the option to select a given characterizer? Was this removed??

Reply:
No, not merged yet.